CONFIG.md

```diff
@@ -484,6 +484,27 @@ should be placed. Defaults to the package root. For Acquia set this to
 **packages.dest.devResources**: Specify where within the package directory the
 `projFiles` should be placed. Defaults to package root.
 
+### Install settings
+
+The `install` task will either install Drupal via the configured `profile`, or if
+a path to a database is configured, load this database:
+
+To specify a profile other than the `standard` profile:
+
+```json
+{
+  "serve": {
```

Comment:
I wonder if we need to combine `serve` settings and the new `install`? I could just add the new db setting to the `serve` section if that is cleaner.